runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -746,7 +746,7 @@ void TR::ARM64PrivateLinkage::buildVirtualDispatch(TR::Node *callNode,
          generateTrg1Src1ImmInstruction(cg(), TR::InstOpCode::sbfmx, callNode, x9, x9, 0x1F); // sxtw x9, w9
          tempMR = new (trHeapMemory()) TR::MemoryReference(vftReg, x9, cg());
          generateTrg1MemInstruction(cg(), TR::InstOpCode::ldroffx, callNode, dstReg, tempMR);
-         gcPoint = generateLabelInstruction(cg(), TR::InstOpCode::b, callNode, vcSnippetLabel, dependencies);
+         gcPoint = generateLabelInstruction(cg(), TR::InstOpCode::b, callNode, vcSnippetLabel);
```

Comment:
Generally, you shouldn't re-use dependencies on different instructions.  To be completely safe you should clone the dependencies here.  Alas, the `clone()` function isn't implemented for AArch64.  In this particular case I think it is safe to re-use in this context, so I'll let this go ahead in this PR.